rls/src/main/java/io/grpc/rls/RlsProtoConverters.java

```diff
@@ -183,7 +176,21 @@ static GrpcKeyBuilder convert(Map<String, ?> keyBuilder) {
             matcher.isOptional(), "NameMatcher for GrpcKeyBuilders shouldn't be required");
         nameMatchers.add(matcher);
       }
-      return new GrpcKeyBuilder(names, nameMatchers);
+      ExtraKeys extraKeys = ExtraKeys.DEFAULT;
+      Map<String, String> rawExtraKeys =
+          (Map<String, String>) JsonUtil.getObject(keyBuilder,  "extraKeys");
+      if (rawExtraKeys != null) {
+        extraKeys = ExtraKeys.create(
+            /* host= */ rawExtraKeys.get("host"),
```

Comment:
nit: I think here arguments are pretty clear from the context, without extra comment clarification.

Reply:
You are right. Done.